docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +386,150 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
 grateful for `this guidance from Ada
 Initiative <http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports>`__.
 
-We really appreciate the work that all of the communities linked above
-have put into creating such a well considered process.
+We highly appreciate the work that all of the communities linked above
+have put into creating code of conduct documents and processes.
 
 This Code of Conduct is licensed under a `Creative Commons Attribution
 4.0 International <https://creativecommons.org/licenses/by/4.0/>`__ (CC
 BY 4.0 CA) license which means you are free to share and adapt the work
-so long as the attribution to `The Turing
-Way <https://github.com/alan-turing-institute/the-turing-way>`__ project
-with Dr Kirstie Whitaker as the principal investigator is retained,
-along with the attribution to the Carpentries, The Alan Turing Institute
-Data Study Group organising team, Citizen Lab and the other resources.
+so long as attribution is maintained to substantial sources of contribution:
+
+* The `Django project <https://www.djangoproject.com/>`__
+* The `Contributor Covenant <http://contributor-covenant.org/>`__,
+* The `Citizen Lab <https://citizenlab.ca/>`__
+* The `Alan Turing Institute Data Study Groups <https://www.turing.ac.uk/collaborate-turing/data-study-groups>`__ organising team
+* `The Turing Way <https://github.com/alan-turing-institute/the-turing-way>`__ project
+* `The Carpentries <https://carpentries.org/>`__
+* The ``sktime`` project
+* other sources and influences mentioned above
+
+
+Appendix A: Guidance in applying the CoC
```

Comment:
I have reviewed until here, will continue another time. Its quite a lot text below which seems to contain quite some controversial sections, so factoring this out might be the best. Especially there is no definition of "resource" and as I wrote already above, why do we need to even have a section on this?

Reply:
Thanks for reading!

I would appreciate if you spell out explicitly whether you yourself approve or disapprove.

E.g., which sentences do *you* (Martin) want to have struck, changed, etc.

Reply:
I will add a definition of "resource" in appendix A